actions/image-build_docs/build_docs.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+################################################################################
+#
+# The master build_docs.sh script lives at
+# NWChemEx-Project/DeveloperTools/.gihub/workflows/scripts. The build_docs.sh
+# script contained in all other NWChemEx-Project repositories is a synchronized
+# copy of the master script. Thus to make changes to the build_docs.sh file
+# please make them to the master script.
+#
+################################################################################
```

Comment:
```suggestion
```

I realize you're copy/pasting these, but may as well clean them up in the process.